src/components/index.jsx

```diff
@@ -32,33 +34,68 @@ function Home() {
 	useEffect(() => {
 		client.get(baseURL).then(setJobs);
 	}, []);
+	const [open, setOpen] = useState(false);
+
 	return (
 		<>
 			<div className="container">
 				<div className="main-body">
 					<h2>Search For Developer Jobs</h2>
 					<div className="main-card">
-						<div>
+						<div className={cn(' form-wrapper', open && 'active')}>
```

Comment:
Remove prepending space on `form-wrapper`

Reply:
I also feel like this should be its own component that includes the button that opens and closes it.

Reply:
i solved this